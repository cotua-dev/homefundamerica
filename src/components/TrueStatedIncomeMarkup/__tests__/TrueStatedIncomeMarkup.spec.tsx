import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { default as TrueStatedIncomeMarkup } from '..';

describe('TrueStatedIncomeMarkup component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<TrueStatedIncomeMarkup/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
