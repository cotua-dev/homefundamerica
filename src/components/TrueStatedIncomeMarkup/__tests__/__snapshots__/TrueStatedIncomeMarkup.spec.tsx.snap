// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrueStatedIncomeMarkup component renders 1`] = `
<section
  class="contentWrapper"
>
  <article>
    <h2>
      What is a True Stated Income Loan?
    </h2>
    <strong>
      In short, a true stated income loans allow borrowers to simply state their monthly income on a mortgage application instead of verifying the actual amount by furnishing pay stubs and/or tax returns.
    </strong>
    <p>
      For self-employed or small business owners looking for a business investment, a true stated income loan might work for you. With a true stated income loan, clients in the program can access any of our flexible down payment options.
    </p>
    <i>
      You are eligible if you have the amount of the loan requested sitting in the bank after the down payment, closing costs, 6 - 12 months PITI (Principle, Interest, Taxes, & Insurance) Reserves on the property, 2 months PITI Reserves on investment properties, and lastly 5 years of revolving debt payments.
    </i>
    <p>
      Loan amounts and assets vary widely, but, to put this into practice, here is an example:
    </p>
    <strong>
      An individual purchasing a home for $1 million with a 20% down payment needs:
    </strong>
    <p>
      <strong>
        $200,000
      </strong>
       as a down payment.
    </p>
    <p>
      <strong>
        PITI Reserves
      </strong>
       of 6 months - $41,000
    </p>
    <p>
      <strong>
        Average Closing Costs
      </strong>
       totaling $24,000
    </p>
    <p>
      <strong>
        Revolving Debts
      </strong>
       of $1,000 per month for 5 years - $60,000
    </p>
    <i>
      The above numbers break down as follow:
    </i>
    <p>
      <strong>
        Total
      </strong>
       $325,000
    </p>
    <p>
      <strong>
        Loan Requested
      </strong>
       $800,000
    </p>
    <p>
      <strong>
        Total Required in the bank or in Other Assets
      </strong>
       $1,125,000
    </p>
    <h2>
      Qualification Criteria
    </h2>
    <p>
      <strong>
        No Employment
      </strong>
       or income is required
    </p>
    <p>
      <strong>
        Credit Score
      </strong>
       of 620
    </p>
    <p>
      <strong>
        DTI
      </strong>
       is not calculated
    </p>
    <p>
      <strong>
        Cash out
      </strong>
       can be used as reserves
    </p>
    <p>
      <strong>
        Up to $3,000,000
      </strong>
       loan amounts
    </p>
    <p>
      <strong>
        90% LTV
      </strong>
       on purchase
    </p>
    <p>
      <strong>
        Interest Only
      </strong>
       option available
    </p>
    <p>
      <strong>
        70% LTV
      </strong>
       on cash out
    </p>
    <p>
      <strong>
        Price
      </strong>
       as full doc or near prime
    </p>
    <p>
      <strong>
        Reserves
      </strong>
       6 months needed
    </p>
    <p>
      Apply Now For True Stated Income
    </p>
  </article>
</section>
`;
